client/blocks/login/social.jsx

```diff
@@ -166,14 +174,14 @@ class SocialLoginForm extends Component {
 	};
 
 	getRedirectUrl = ( uxMode, service ) => {
-		return uxMode
-			? `https://${ ( typeof window !== 'undefined' && window.location.host ) +
-					login( { isNative: true, socialService: service } ) }`
-			: null;
+		const host = typeof window !== 'undefined' && window.location.host;
+		return `https://${ host + login( { isNative: true, socialService: service } ) }`;
```

Comment:
This appears to ignore `uxMode` now, for every service (not just Apple). Is this intentional? If so, and if it's not needed for anything, we should drop the parameter to avoid confusion.

Reply:
Yes, I wanted to give the redirect uri to both login buttons in all cases, even when the uxMode is `popup`. We might need it anyway to initialize the client (the apple client for instance will throw if we call init without a redirect uri).